.github/workflows/lint.yml

```
@@ -10,18 +10,42 @@ jobs:
+          pipx install poetry==1.8.5 --python ${{ steps.setup-python.outputs.python-path }}
+
+      - name: Check python version
+        run: |
+          poetry run python --version
+
+      - name: Setup a local virtual environment
+        run: |
+          poetry config virtualenvs.create true --local
+          poetry config virtualenvs.in-project true --local
+
+      - uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)